src/assets/icons/electronics/electronics.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { SVGProps } from 'react';
+
+const Electronics = (props: SVGProps<SVGSVGElement>) => (
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		fill="none"
+		viewBox="0 0 53 52"
+		{...props}
+	>
+		<g clipPath="url(#a)">
+			<rect width={32} height={44} x={10.5} y={4} fill="#1E4C2F" rx={6} />
+			<circle cx={26.5} cy={41} r={3} fill="#fff" />
+		</g>
+		<defs>
+			<clipPath id="a">
```

Comment:
`id` attribute for any svg file should be unique. Please check every icon which is built this way and update id attributes relatively.

Also, did you use [svgomg.net](https://svgomg.net/) before using SVGR? Because I would expect these paths to be merged into one single path 🤔 
Please check all the icons and verify that they are optimized with svgomg before converted into components with svgr.